data/json/items/vehicle/cables.json

```diff
@@ -18,6 +18,25 @@
     "use_action": "CABLE_ATTACH",
     "flags": [ "CABLE_SPOOL" ]
   },
+  {
+    "type": "TOOL",
+    "id": "tow_cable",
+    "name": "tow cable",
+    "description": "A tow cable, attached to a vehicle, it could be used to pull another vehicle.",
+    "to_hit": -1,
+    "color": "light_blue",
+    "symbol": "&",
+    "material": [ "steel", "plastic" ],
+    "volume": "3000 ml",
+    "weight": "875 g",
```

Comment:
Since we're not doing weight limits yet, this should be 24 kgs of chain and cable with a volume of 3L or so.